build/android/src/main/res/layout/assetcopy.xml

```diff
@@ -18,7 +18,7 @@
         android:ellipsize="middle" 
         android:singleLine="true"
         android:layout_gravity="center_horizontal"
-        android:text="preparing media ..."
+        android:text="@string/preparing_media"
```

Comment:
Disabling the lint will disable it for all instances, not just this one.  I've already translated it (and other strings) into Spanish in a commit I'm working on anyways.